### PR TITLE
Adds chaos monkey setup

### PR DESCRIPTION
**Pull request by marcingrzejszczak**
related https://github.com/spring-petclinic/spring-petclinic-microservices/pull/177

### CODE REVIEW

**Review by jonatan-ivanov — APPROVED** [review]

**Comment by arey** [comment]
Sorry I missed this PR. I will check

File: application.yml [hunk 1]
```diff
@@ -54,6 +54,19 @@ management:
       prometheus:
         enabled: true
 
+# Chaos Engineering
```
Comment (arey) [inline application.yml:57]: The `run_all.sh` script starts the Spring Boot app with the profile `--spring.profiles.active=chaos-monkey`. Thus I propose to move this config section in a dedicated one like for `mysql`: 
```
spring:
  config:
    activate:
      on-profile: chaos-monkey
```

File: application.yml [hunk 2]
```diff
@@ -54,6 +54,19 @@ management:
...
+management.endpoint.chaosmonkey.enabled: true
+chaos:
+  monkey:
+    enabled: true
+    watcher:
+      component: false
+      controller: false
+      # Defaults to potential issues with repositories - can be changed via REST API
```
Comment (arey) [inline application.yml]: Does this issue is referenced in the Chaos Monkey GitHub project?

File: application.yml (hunk 1 above)
Comment (marcingrzejszczak) [reply application.yml:57]: Sure no problem

File: application.yml (hunk 2 above)
Comment (marcingrzejszczak) [reply application.yml]: Ah it's a copy paste from my pet project, I'll remove it

**Comment by marcingrzejszczak** [comment]
Updated the YAML

File: application.yml [hunk 3]
```diff
@@ -54,6 +54,23 @@ management:
...
+    activate:
+      on-profile: chaos-monkey      
+management.endpoint.chaosmonkey.enabled: true
+chaos:
+  monkey:
+    enabled: true
+    watcher:
+      component: false
+      controller: false
+      repository: true
```
Comment (arey) [inline application.yml]: Do you know why the repository watcher is set to true?

Comment (marcingrzejszczak) [reply application.yml]: That's a sensible default but we can set it to `false`

Comment (marcingrzejszczak) [reply application.yml]: Converted it to `false`

**Pull request merged**